From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@CawaMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
automation : rbac updates

mimig1 committed:
Cosmos DB: TOC addition for Kafka article

paulathurman committed:
edit pass: site-recovery-articles-batch1

mimig1 committed:
changed to 2 line version

mimig1 committed:
adding contextual toc

mimig1 committed:
correcting contextual toc link

ggailey777 committed:
Draft of version note

ggailey777 committed:
Fixed minor version

ggailey777 committed:
Remove the v2 note from Cosmos DB since v2 also requires core tools.

timeyoutakeit committed:
Merge remote-tracking branch 'MicrosoftDocs/master'

timeyoutakeit committed:
Reviewing Acrolinx suggestions to improve docs

Commit:
updates to rbac

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into site-recovery-articles-batch1

mimig1 committed:
added azure to the link

paulathurman committed:
edit pass: site-recovery-articles-batch1

paulathurman committed:
edit pass: site-recovery-articles-batch1

mimig1 committed:
Merge branch 'master' into 01222018toc

mimig1 committed:
Update TOC.yml

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

ggailey777 committed:
Complete draft of VS Code QS

ggailey777 committed:
Merge branch 'vscode' of github.com:ggailey777/azure-docs-pr into cli

ggailey777 committed:
In-progress updates

Commit:
updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rbacguide

Commit:
updates

Commit:
updates

Commit:
updates

ausfeldt committed:
add 1.1.0 to release notes

@mimig1

Commit:
updates

Commit:
updates

ausfeldt committed:
fix date

@mimig1

Commit:
removed note:

Commit:
image updates

Commit:
updates

ggailey777 committed:
Fixed broken anchor links

giladmit committed:
small clarifications

Commit:
narayan feedback

Commit:
updates

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ikev2

alkohli committed:
migrate StorSimple Support options from MSDN

cherylmc committed:
update

alkohli committed:
acrolinx changes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into imageupdates

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
update

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into site-recovery-articles-batch1

paulathurman committed:
edit pass: site-recovery-articles-batch1

johnkemnetz committed:
Update monitoring-activity-log-schema.md

cherylmc committed:
update

Juliako committed:
removed the mentioning of the Publisher portal

Juliako committed:
removed the mentioning of the Publisher portal

Juliako committed:
removed the mentioning of the Publisher portal

Juliako committed:
removed the mentioning of the Publisher portal

JYOTHIRMAISURI committed:
new images for unified alerts

JYOTHIRMAISURI committed:
activity log (preview) contents and the new webhook article from Vijay

JYOTHIRMAISURI committed:
fixing broken links in new webhook article

wh-alice committed:
fix cli link in virtual machine folder starting from a to c

daveirwin1 committed:
Updates for 9.4 agent

Commit:
updates

Commit:
updateS

davidmu1 committed:
new internal redirection tutorial

Commit:
updates

Commit:
Add link to the Architecture Center landing page

davidmu1 committed:
minor correction

ggailey777 committed:
Removed new VS Code QS topic from the CLI PR

Commit:
updates

JYOTHIRMAISURI committed:
incorporating the PR review

CarlRabeler committed:
priv

alkohli committed:
Modified the TOC to include article in Resources

v-shils committed:
Merge pull request #32775 from johnkemnetz/patch-1

Update monitoring-activity-log-schema.md

GitHubber17 committed:
Merge pull request #32015 from ggailey777/cli

Draft of version note for CLI scripts topics

rmca14 committed:
Merge pull request #32079 from asb3993/amburns-mobile-auth-improvements

Mobile auth improvements

JYOTHIRMAISURI committed:
incorporating PR review

alkohli committed:
Non-blocking issue fixed

dlepow committed:
Batch VNet article updates

v-shils committed:
Merge pull request #32809 from wh-alice/virtualmachinefolder

fix cli link in virtual machine folder starting from a to c

alkohli committed:
fixed the broken link

JYOTHIRMAISURI committed:
format fix

davidmu1 committed:
new url redirect tutorial

alkohli committed:
fixed broken link 2

HJToland3 committed:
FAQ question add

davidmu1 committed:
unneeded article

PRMerger16 committed:
Merge pull request #32834 from giladmit/GiladWorkingBranch

small clarifications

Commit:
updates

sethmanheim committed:
Freshness updates

PRMerger20 committed:
Merge pull request #32836 from HJToland3/master

FAQ question add

ktoliver committed:
Merge pull request #32803 from JYOTHIRMAISURI/master

new images for unified alerts

msangapu-msft committed:
Added 2.0 to .net version

Added 2.0 to .net version

Commit:
updates

PRMerger19 committed:
Merge pull request #32838 from sethmanheim/wk-fresh

Freshness updates

Commit:
freshness update

davidmu1 committed:
new url redirect tutorial

tfitzmac committed:
remove includes used only once

alkohli committed:
removed extra headers

GitHubber17 committed:
Merge pull request #32822 from davidmu1/tutorialintred

new internal redirection tutorial

daveirwin1 committed:
Updated diagram with latest OS support

GitHubber17 committed:
Merge pull request #32501 from georgewallace/rbacguide

automation : update to rbac guide

carolinacmoravia committed:
M22734: Possible typo mistake: "and, or" 

Hello, @mattbriggs, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

davidmu1 committed:
Merge branch 'master' into tutorialurlred

Commit:
Merge pull request #32841 from ahgyger/studio_arm

freshness update

bandersmsft committed:
cost mgt - adding connect Google account

carolinacmoravia committed:
M22735: Missing preposition: "in"

bandersmsft committed:
Acrolinx fixes

v-shils committed:
Merge pull request #32821 from georgewallace/imageupdates

automation:  Image updates #5

christiankuhtz committed:
workaround for az cli 2.0 bug

az feature show is missing from current az cli 2.0 build. fix expected in az cli 2.0 2/14 release

carolinacmoravia committed:
M22736: "Date-Type Type" should be "Date-Time Type"

Hello, @ecfan, 

Translator has reported possible source content issue. It seem that "Date-Type Type" should be "Date-Time Type"

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

mmacy committed:
[ACR] add Java ref to TOC and LP

christiankuhtz committed:
added json output az cli 2.0 commands

kgremban committed:
Polishing LBS docs

Commit:
Merge pull request #32846 from carolinacmoravia/patch-598

M22734: Possible typo mistake: "and, or"

davidmu1 committed:
new http redirect tutorial

dlepow committed:
Incorp'd Rohit's feedback

wesmc7777 committed:
User story 1077069 Connection blips may occur during scaling

dlepow committed:
Wording update

v-shils committed:
Merge pull request #32736 from cherylmc/ikev2

update

v-shils committed:
Merge pull request #32827 from MikeWasson/wasson/architecture-pivot

Add link to the Architecture Center landing page

v-shils committed:
Merge pull request #32831 from CarlRabeler/dani2

priv

PRMerger13 committed:
Merge pull request #32849 from christiankuhtz/patch-48

workaround for az cli 2.0 bug

v-shils committed:
Merge pull request #32845 from daveirwin1/master

Updates for agent ver 9.4.0

mimig1 committed:
Added maintainContext: true

Ja-Dunn committed:
Merge pull request #32189 from paulathurman/site-recovery-articles-batch1

Edit pass: Site recovery articles batch1

PRMerger15 committed:
Merge pull request #32853 from kgremban/feb5-lbs

Polishing LBS docs

wesmc7777 committed:
Style and grammar updates

v-shils committed:
Merge pull request #32779 from Juliako/APIMUpdates

removed the mentioning of the Publisher portal

Ja-Dunn committed:
Merge pull request #32842 from tfitzmac/0205includes

remove includes used only once

PRMerger17 committed:
Merge pull request #32858 from msangapu/msangapu-patch-2

Added 2.0 to .net version

PRMerger14 committed:
Merge pull request #32857 from wesmc7777/master

User story 1077069 Connection blips may occur during scaling

Ja-Dunn committed:
Merge pull request #32847 from bandersmsft/master

cost mgt - adding connect Google account

Lisaco88 committed:
Merge pull request #32837 from davidmu1/tutorialurlred

Tutorialurlred

Ja-Dunn committed:
Merge pull request #32743 from alkohli/SSSupport

Migrate StorSimple Support options from MSDN

Commit:
Merge pull request #32833 from dlepow/fix

Batch VNet article updates

Ja-Dunn committed:
Merge pull request #32851 from mmacy/acr-java-ref-add

[ACR] add Java ref to TOC and LP

davidmu1 committed:
new http redirection tutorial

Commit:
Merge pull request #32848 from carolinacmoravia/patch-599

M22735: Missing preposition: "in"

davidmu1 committed:
Merge branch 'master' into tutorialhttpred

PRMerger17 committed:
Merge pull request #32850 from carolinacmoravia/patch-600

M22736: "Date-Type Type" should be "Date-Time Type"

PRMerger18 committed:
Merge pull request #32585 from ausfeldt/patch-1

add 1.1.0 to release notes

Commit:
quickfix01

Ja-Dunn committed:
Merge pull request #31873 from mimig1/01222018toc

Cosmos DB: TOC addition for Kafka article

christiankuhtz committed:
added reference for metrics

Ja-Dunn committed:
Merge pull request #32856 from davidmu1/tutorialhttpred

new http redirect tutorial

Commit:
Merge pull request #32864 from MarkusVi/quickfix01

quickfix01

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

anavinahar committed:
Update azure-virtual-network-limits.md

PRMerger13 committed:
Merge pull request #32865 from christiankuhtz/patch-49

added reference for metrics

v-alje committed:
Merge pull request #32866 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger20 committed:
Merge pull request #32867 from anavinahar/patch-36

Update azure-virtual-network-limits.md
